ads/google/a4a/utils.js

```diff
@@ -113,6 +114,27 @@ export const TRUNCATION_PARAM = {name: 'trunc', value: '1'};
 const CDN_PROXY_REGEXP =
   /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org((\/.*)|($))+/;
 
+/** @const {string} */
+const TOKEN_VALUE =
+  'A8Ujr8y+9sg/ZBmCs90ZfQGOUFJsAS/YaHYtjLAsNn05OaQXSmZeRZ2U1wAj3PD74WY9re2x/TwinJoOaYuqFQoAAACBeyJvcmlnaW4iOiJodHRwczovL2FtcHByb2plY3QubmV0OjQ0MyIsImZlYXR1cmUiOiJDb252ZXJzaW9uTWVhc3VyZW1lbnQiLCJleHBpcnkiOjE2MzE2NjM5OTksImlzU3ViZG9tYWluIjp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9';
+
+/**
+ * Inserts origin-trial token for `attribution-reporting` if not already
+ * present in the DOM.
+ * @param {!Window} win
+ */
+export function maybeInsertOriginTrialToken(win) {
+  if (win.__AMP_OT_INSERTED) {
+    return;
+  }
+  const metaEl = createElementWithAttributes(win.document, 'meta', {
```

Comment:
How about checking `win.document.querySelectorAll('meta[http-equiv="origin-trial"])`?

Reply:
I did `content=TOKEN_VALUE` just in case there is another origin trial token, but done.